server/models.py

```diff
@@ -1,13 +1,15 @@
 from datetime import datetime
 from clear_my_record_backend.server import dbs
 
-class Qualifying_Questions(dbs.Model):
+
+class Qualifying_Question(dbs.Model):
```

Comment:
We'll probably want an accompanying schema migration. 

You can do that with

```
flask db migrate -m "some message"
```

if I'm remembering this correctly.

Reply:
should i run that and just add it as its own commit in this pr?

Reply:
yes.